core/domain/wipeout_service_test.py

```diff
@@ -4894,3 +4894,156 @@ def test_multiple_blog_post_models_are_pseudonymized(self):
         for blog_post_rights_model in blog_post_rights_models:
             self.assertTrue(
                 self.user_1_id not in blog_post_rights_model.editor_ids)
+
+
+class PendingUserDeletionTaskServiceTests(test_utils.GenericTestBase):
+    """Provides testing for the delete users pending to be deleted taskqueue
+    service methods of wipeout service."""
```

Comment:
```suggestion
    service methods of wipeout service.
    """
```

The terminating `"""` should be on its own line. Ditto for the docstring below.

Reply:
Done.